site/migration/01-migration.md

```diff
@@ -0,0 +1,257 @@
+the functionality. Here are the one-liners for each command group in kpt `v1.0`.
+
+1. `pkg:` Commands for composing and describing packages.
+2. `fn:` Commands for running functions for validation and customization.
+3. `live:` Commands for interacting with a live cluster.
+
+`PKG_PATH vs DIR:` Most of the commands accept only `PKG_PATH` as input which means
+the input directory must have a valid Kptfile. Few commands can work on just the
+simple `DIR` with resources. Commands operate on the current working directory by default.
+
+| [Existing Commands]                                                                              | [New Commands]                                                                                                                                                                                                                                   |
+| ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] LOCAL_DEST_DIRECTORY [flags]                       | kpt pkg get REPO_URI[.git]/PKG_PATH[@VERSION] [flags] [LOCAL_DEST_DIRECTORY] <br> Fetch a remote package from a git subdirectory and writes it to a new local directory.                                                                         |
+| kpt pkg init DIR [flags]                                                                         | kpt pkg init [DIR] [flags] <br> Initializes an existing empty directory as a kpt package by adding a Kptfile.                                                                                                                                    |
+| kpt pkg update LOCAL_PKG_DIR[@VERSION] [flags]                                                   | kpt pkg update [PKG_PATH][@version] [flags] <br> Pulls in upstream changes and merges them into a local package.                                                                                                                                 |
+| kpt pkg fix DIR [flags]                                                                          | kpt fn eval --image gcr.io/kpt-fn/fix:v0.1 <br> Fix a local package which is using deprecated features.                                                                                                                                          |
+| kpt pkg desc DIR/ [flags]                                                                        | Deprecated: read Kptfile directly                                                                                                                                                                                                                |
```

Comment:
slight language change "Deprecated in favor of reading Kptfile directly" seems less of a command.

Reply:
Update.